setup.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
```

Comment:
Can you add this script with a default settings to a separate job on Github Actions ?
This way we will have some degree of knowledge whether it always works